src/components/Footer/Footer.tsx

```diff
@@ -19,11 +19,19 @@ export default function Navbar() {
         <Styled.ContactWrapper>
           <Markdown>{footerContent}</Markdown>
           <br />
+          <a href={'http://creativecommons.org/licenses/by/4.0/'}>
```

Comment:
Minor thing, but should be https to be nice to CC?

Reply:
Nice, thanks